app/models/mission.rb

```diff
@@ -54,15 +54,29 @@ def java_main_contents
   end
 
   def java_problem_contents(user_id)
-    submit = Submit.where(user_id: user_id, mission_id: self).last
-    java_contents = if submit
-                      java_submitted_contents(submit.id)
+    submit_id = Submit.last_question_submit_ids(
+      user_id, Question.where(problem_id: problems)
+    ).values
+    java_contents = if submit_id
+                      java_present_contents(submit_id)
                     else
                       java_pierced_contents
                     end
     java_contents
   end
 
+  def java_present_contents(submit_id)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/AbcSize: Assignment Branch Condition size for java_present_contents is too high. [27.18/15]